packages/nimble-components/src/table-column/text/index.ts

```diff
@@ -1,17 +1,8 @@
-import { attr } from '@microsoft/fast-element';
 import { DesignSystem } from '@microsoft/fast-foundation';
```

Comment:
What do you think about keeping it all in a single file?
https://github.com/ni/nimble/compare/table-column-width-api-updated...mraj-table-column-width-api-updated?expand=1

Have to disable the multiclass rule but that's not so bad. Things I was concerned about is I don't think the middle -Mixin class was necessary, I think it would make the pattern confusing if you have multiple mixins for a class. We also export fewer intermediate / utility classes this way (just TableColumnText is exported from index).